helm/secrets-provider/templates/secrets-provider.yaml

```diff
@@ -65,9 +59,25 @@ spec:
         - name: CONJUR_AUTHN_LOGIN
           value:  {{ .Values.environment.conjur.authnLogin | quote }}
 
+        - name: SECRETS_DESTINATION
+          value: k8s_secrets
+
+        - name: CONTAINER_MODE
+          value: application
+
         - name: K8S_SECRETS
           value: {{ .Values.environment.k8sSecrets | join "," }}
 
+        {{- if .Values.environment.conjur.syncIntervalTime }}
+        - name: SYNC_INTERVAL_TIME
+          value: {{- .Values.environment.conjur.syncIntervalTime }}
+        {{- end }}
+
+        {{- if .Values.environment.conjur.retryCount }}
+        - name: RETRY_COUNT
```

Comment:
how about `RETRY_COUNT_LIMIT`? or something like that. `count` sounds like an ongoing procedure